### ISSUE TITLE
django 4 'cannot import name url' from 'django.conf.urls'

### ISSUE THREAD

**Issue by ekramer121** [nnseva/django-taggit-bulk#2]
I am using Django 4.0.6 and am trying to use the taggit_bulk package. Unfortunately when I go to import your urls.py I cannot use django.conf.urls.url as that was deprecated in 3 and removed for 4.0 of Django.

using re_path in my own urls allows me to import xxxx but when it gets to line 11 in taggit_bulk/urls.py there is an exception of:
```

  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "D:\Projects\Python\dr_mercantile_exchange\venv\lib\site-packages\taggit_bulk\urls.py", line 4, in <module>
    from django.conf.urls import url
ImportError: cannot import name 'url' from 'django.conf.urls' (D:\Projects\Python\dr_mercantile_exchange\venv\lib\site-packages\django\conf\urls\__init__.py)

```

I tried to import your urls directly from my app but that lead to an even stranger url saying that taggit_bulk was not a registered namespace, which of course it was. I am fairly new to Django but I think all you need to do is change

from django.conf.urls import url
urlpatterns = [
    url(r'^wizard/$', TaggitBulkWizard.as_view(), name='wizard'),
]

to 

from django.urls import re_path
urlpatterns = [
    re_path(r'^wizard/$', TaggitBulkWizard.as_view(), name='wizard'),
]

**Comment by nnseva** [comment]
I didn't implement Django 4.0 support yet. If you do, feel free to make a PR. Don't forget to modify `setup.py`, `.travis.yml`, and `tox.ini` files accordingly. Notice that the Django Taggit package has complex dependencies from Django and Python versions, so we need to fix installed versions for different environments.

**Comment by nnseva** [comment]
fixed in 7d8b8c6
